mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -229,13 +231,13 @@ func (e *Ephemeral) processAttributeManifests(ctx context.Context) map[string]*c
 		for _, v := range info.AttributeManifests {
 			for an, at := range v.Attributes {
 				attrs[an] = at
-				stats.Record(ctx, monitoring.AttributesTotal.M(1))
```

Comment:
Do I understand it correctly, that this was incorrect before since it would collect "1" instead of the total count?